src/locales/de.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Start": "Starten",
```

Comment:
Only thing here is that keys should be really keys not translation. So here you have key:value pair. So keys should be named as variables using english (no spaces, upercases but camel_case)

"start": "Starten",
"machine_learning": "Maschinelles Lernen",
"page_not_founr": "Entschuldigung, die angeforderte Seite konnte nicht gefunden werden. Versuchen Sie, zurück nach Hause zu navigieren.",